text/0000-docs-information-architecture

```diff
@@ -0,0 +1,154 @@
+    - 6. Source plugins and rendering queried data
+    - 7. Transformer plugins and transforming data into site content
+    - 8. Programmatically creating pages from data
+- Adding images to your site (or "Adding svg or blur-up effect to images)
+- Creating your own plugin (tutorial for creating a plugin we've already vetted as easy to create)
+- E-commerce
+- User authentication
+
+Reference docs
+- How Gatsby works
+    - The Gatsby core philosophy
+    - Basic structure of a Gatsby site (includes config.js)
+    - How Gatsby works with GraphQL (includes Node interface and how GraphQL works)
+    - How Gatsby works with React (API specification)
+    - How Gatsby works with Redux (bound action creators)
+    - How Gatsby works with plugins
+    - PRPL pattern
+    - Lifecycle APIs
+- API references
+    - @jlengstorf this idea is from you, don't know how to fill this in...
+    - API specification
+    - Gatsby Browser APIs
+    - Gatsby Node APIs
+    - Gatsby SSR APIs 
+
+- Gatsby CLI commands
+- Basic tasks
+    - Setting up dev environment   
+        - Gatsby on Windows
+    - Customizing site structure
+        - html.js
+        - webpack config
+        - Environment variables (right place for this??)
+    - Adding content 
+        - Images
+        - Markdown pages
+        - adding tags and categories to blogposts
+        - adding a 404 page
+    - Querying for data with GraphQL
+    - Debugging
+        - Debugging HTML builds
+        - Debugging Replace Renderer API
+    - Adding dynamic functionality
+        - Building apps with Gatsby
+        - E-commerce
+        - authentication
+        - dynamically rendered navigation
+    - Deploying
+    - Hosting
+        - Path Prefix
+    - Performance
+        - Search Engine Optimization (SEO)
+        - Browser 
+        - Setting up proper HTTP caching
+- Migration Guides
+    - v0 to v1
+    - v1 to v2
+    - v0 to v2 (good idea??)
+- Gatsby components and libraries
+    - Glamor
+    - Styled components
+    - gatsby-image
+
+
+Plugins
+- Plugin Authoring Guides
+    - How Gatsby works with plugins
+    - Plugin library
+    - When to create your own plugin?
+    - Source plugins
+    - Transformer plugins
+    - All other plugins
+    - How to get your plugin to show up in the Plugin Library
+
+
+Contributing
+- Welcome to Gatsby's open source community!
+- Code of Conduct
+- How to Contribute
+    - List of ways to contribute
+    - How to file an issue
+    - How to submit a PR
+- Style Guides
+    - Written documents
+    - Visual design
+- Templates & Resources
+    - Workshops
+    - PowerPoint presentations
+    - Blogpost
+    - Tutorial
+    - Plugin READme
+    - Site READme
+
+
+
+What we may not need to include in the docs:
+- Starters and the Awesome Gatsby stuff will appear in a starter library and a site showcase, so will no longer need to be in docs fairly soon.
+- Integration tutorials and guides (such as Gatsby + Netlify, Gatsby + Contentful). 
```

Comment:
This new IA with an extra level would actually make it easier to include tutorials & docs, eg under Docs/Deploying for Netlify, or under Tutorials for Contentful. "Building a Gatsby + Contentful site"

Reply:
True! I remember @jlengstorf thinking of some reasons not to include this kind of stuff in core docs, I think mainly because it can be unwieldy to maintain. Jason do you remember more reasons?

Reply:
If we can add clear and encouraging instructions on how to submit updates to the guides, maybe we can rely on the community to keep them updated.

Reply:
@m-allanson yes, and perhaps some sort of spring cleaning schedule could help. Like a docs Friday once a month where anyone who updates a doc can get a sticker or something.